extensions/shared/register-jetpack-block.js

```diff
@@ -57,11 +57,10 @@ function buildBlockTags( name ) {
  * @returns {string} Block title
  */
 function buildBlockTitle( blockTitle, blockTags = [] ) {
-	if ( blockTags.length ) {
-		blockTitle = `${ blockTitle } (${ blockTags.join( ', ' ) })`;
```

Comment:
it isn't related to the fix, but I think reassigning a value to the argument is something that we'd like to avoid too.

Reply:
Agreed, this looks like a good change to make while we're here